src/popup/router/components/TransactionItem.vue

```diff
@@ -3,45 +3,82 @@
     class="transaction-item"
     :to="{ name: 'tx-details', params: { hash: transaction.hash } }"
   >
-    <div class="left">
-      <Pending
-        v-if="transaction.pending"
-        class="icon"
-      />
-      <TokenAmount
-        :amount="getTxAmountTotal(transaction)"
-        :symbol="getTxSymbol(transaction)"
-        :aex9="isTxAex9(transaction)"
-        :direction="getTxDirection(transaction)"
-        :hide-fiat="showType"
-        data-cy="amount"
-      />
-      <span
-        v-if="showType"
-        class="type"
-      >
-        ({{ ($t('transaction.type')[txType]) || txType }})
-      </span>
+    <div class="header">
+      <span />
+      <div class="status">
+        <template v-if="transaction.pending">
+          <Pending class="icon" />
+          <span class="pending">Pending...</span>
+        </template>
+        <template v-else>
+          <span data-cy="date">{{ transaction.microTime | formatDate }}</span>
+          <span data-cy="time">{{ transaction.microTime | formatTime }}</span>
+          <div
+            v-if="isErrorTransaction"
+            class="error"
+          >
+            <Warning
+              v-if="transaction.tx.returnType === 'abort'"
+              class="icon"
+            />
+            <Reverted
+              v-else
+              class="icon"
+            />
+            <span>{{ $t('transaction.returnType')[transaction.tx.returnType] }}</span>
+          </div>
+        </template>
+      </div>
+      <span />
     </div>
-    <div
-      v-if="!transaction.pending"
-      class="right"
-    >
-      <span data-cy="date">{{ transaction.microTime | formatDate }}</span>
-      <span data-cy="time">{{ transaction.microTime | formatTime }}</span>
+    <div class="body">
+      <div
+        v-for="token in tokens.filter(({ amount }) => amount)"
+        :key="token.symbol"
+        :class="['token-row', token.isReceived ? 'received': 'sent', { error: isErrorTransaction }]"
+      >
+        <span class="amount">
+          {{
+            `${token.isReceived ? '+' : '-'}
+            ${amountRounded(token.decimals
+            ? convertToken(token.amount || 0, -token.decimals) : token.amount)}`
+          }}
```

Comment:
wouldn't it be possible to use TokenAmount.vue here?

Reply:
No, because the fiat value of the tokens should be shown in the `footer`, despite it appearing in the rows. Plus having TokenAmount.vue shoud be rewritten an include way more logic that it have right now. I would rather do it when we are agreed on changing logic in it in several or all places. 
We can mark that with `TODO: extract logic into TokenAmount` for now.